source/collections/clustered-collection.txt

```diff
@@ -0,0 +1,59 @@
+.. _clustered-collection:
+
+=============================
+Create a Clustered Collection
+=============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+:manual:`Clustered collections </core/clustered-collections/>` are collections 
+with a clustered index. Clustered collections store documents ordered by :manual:`clustered index </reference/method/db.createCollection/#std-label-db.createCollection.clusteredIndex>` 
+key value.
```

Comment:
(praise & suggestion)
This is a really nice first paragraph. I love that you're very briefly explaining a clustered collection and linking out to more information.

What do you think about adding one additional brief sentence about when/in what circumstances clustered collections are a good choice?

Reply:
Done.